cmd/cluster-node-tuning-operator/main.go

```diff
@@ -1,64 +1,149 @@
+/*
```

Comment:
Let's remove the license text in this file too please.  I believe the `LICENSE` file is enough.